scribblings/libgit2.scrbl

```diff
@@ -1,8 +1,32 @@
 #lang scribble/manual
 
-@title[#:style 'toc]{Functions}
+@title{Libgit2: Bindings for the libgit2 git library}
 
-@local-table-of-contents[]
+@author[(author+email "Brad Busching" "bradley.busching@gmail.com")]
+
+@(require "doc.rkt")
+
+@defmodule[libgit2]
+
+libgit2 is a portable, pure C implementation of the Git
+core methods provided as a re-entrant linkable library with a solid API,
+allowing you to write native speed custom Git applications in any language
+which supports C bindings.
+
+The @racketmodname[libgit2] Racket bindings begaas a senior project
+at California Polytechnic State University, San Luis Obispo
+under the guidance of John Clements (clements[at]brinckerhoff.org)
```

Comment:
@jbclements would you prefer for this to be obfuscated or a link?

Reply:
doesn't need to be obfuscated, but I would probably use clements@racket-lang.org in this setting.